doc/api/buffer.md

```diff
@@ -3768,6 +3768,10 @@ console.log(buf2.toString(undefined, 0, 3));
 // Prints: té
 ```
 
+For encoding and decoding large strings to/from `utf-8`,
+it is recommended to use the [`util.TextEncoder`][]
```

Comment:
Could you include an example right below?

Reply:
sure adding an example

Reply:
Added an example could you check?

Reply:
Could you include an example that uses Buffer? We need to explain how to use TextEncoder to encode-decode buffer strings utf-8

Reply:
Correct. Basically showing how someone would use the Buffer + TextEncoder to improve the performance, instead of toString().

Reply:
Ah ok understood, I guess will update once there is proper consensus that it indeed would be the fastest as @tniessen pointed out